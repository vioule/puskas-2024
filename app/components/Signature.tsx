import Link from "next/link";

export default function Signature() {
  return (
    <Link
      className="text-white absolute bottom-4 right-2 rotate-[-90deg] origin-bottom-left translate-x-[100%] font-roboto uppercase tracking-[0.3rem] max-sm:text-[8px] text-[10px] font-thin opacity-70 mix-blend-difference hover:opacity-100"
      href="https://github.com/vioule"
      target="_blank"
      rel="author"
    >
      <span>design & development - Cano Vincent</span>
    </Link>
  );
}
