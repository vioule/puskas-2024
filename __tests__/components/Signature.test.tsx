import { render } from "@testing-library/react";
import Signature from "@/app/components/Signature";

test("render Signature unchanged", () => {
  const { container } = render(<Signature />);
  expect(container).toMatchSnapshot();
});
