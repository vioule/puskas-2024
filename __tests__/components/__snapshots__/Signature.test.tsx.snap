// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Signature unchanged 1`] = `
<div>
  <a
    class="text-white fixed bottom-4 right-2 rotate-[-90deg] origin-bottom-left translate-x-[100%] font-roboto uppercase tracking-[0.3rem] max-sm:text-[8px] text-[10px] font-thin opacity-70 mix-blend-difference hover:opacity-100 z-[1000]"
    href="https://github.com/vioule"
    rel="author"
    target="_blank"
  >
    <span>
      design & development - Cano Vincent
    </span>
  </a>
</div>
`;
